change/@internal-react-components-5c4e27c8-a1aa-4f57-a5fa-3176b703b9ad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add error bar to show file download error message in the chat screen.",
```

Comment:
`chat screen` for `react-components` package, there is no such thing as chat screen.
Let's use MessageThread or the appropriate component name that we updated.

Reply:
Done